.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2.1
+
+parameters:
+  node_version:
+    type: string
+    default: '18.8'
+
+orbs:
+  aws-cli: circleci/aws-cli@3.1.4
+  deployment-tooling: geckoboard/deployment-tooling@1
```

Comment:
This is a public repo so we should probably think about an alternative CI setup.

Reply:
Yes. We're just using this orb for caching npm install I believe. Given that I don't imagine we need to optimise that much,  we should be fine with a regular `npm install`